tests/null_test.py

```diff
@@ -1,5 +1,5 @@
-from spyql.nulltype import *
-from spyql.sqlfuncs import *
```

Comment:
wildcard imports are not recommended in PEP8.
cf. https://peps.python.org/pep-0008/#imports

Reply:
OK. I must confess that I don't like the idea of maintaining the `spyql.sqlfuncs` list of functions, which is expected to grow. But I guess that's not a problem right now :-)